draft-ietf-httpbis-semantics-latest.xml

```diff
@@ -2681,12 +2681,18 @@ Host: www.example.org
    messages, while others use an explicit identifier.
 </t>
 <t>
-   Responses (both <x:ref>final</x:ref> and <x:ref>interim</x:ref>) can be sent
-   at any time after a
+   All responses, regardless of the status code (including <x:ref>interim</x:ref>
+   responses) can be sent (and be completed) at any time after a
```

Comment:
Lots of parentheses here.  Not sure what to do about that, as they are all useful.  I would probably just use more sentences 🤷‍♂️